lib/pages/twake_welcome/twake_welcome.dart

```diff
@@ -79,15 +84,40 @@ class TwakeWelcomeController extends State<TwakeWelcome> with ConnectPageMixin {
     handleTokenFromRegistrationSite(matrix: matrix, uri: uri);
   }
 
-  void onClickCreateTwakeId() {
+  void onClickCreateTwakeId() async {
     Logs().d(
       "TwakeIdController::onClickCreateTwakeId: Signup Url - $signupUrl",
     );
+    final homeserverExisted = await _homeserverExisted();
+    if (homeserverExisted) return;
     _redirectRegistrationUrl(signupUrl);
   }
 
+  Future<bool> _homeserverExisted() async {
+    if (widget.arg?.isAddAnotherAccount == false) return false;
+    final allHomeserverLoggedIn = (await ClientManager.getClients())
+        .map((client) => client.homeserver.toString())
+        .toList();
+    Logs().i('All homeservers: $allHomeserverLoggedIn');
+    final homeserverExists = allHomeserverLoggedIn.any(
+      (homeserver) => "$homeserver/".contains(AppConfig.homeserver),
```

Comment:
can you explain this please ?

Reply:
When use add another account with TWP. I get all current `homeserver` and  check if `homeserver` existed and logged.
  - If `homeserver` existed → Can't support login 2 accounts with 1 `homeserver`
   example: `homeserver` logged = ['domain1', 'domain2'] and click on button login with 'domain1' → return error
  - If `homeserver` does not exist → continue